indico_toolkit/indico_wrapper/modelop.py

```diff
@@ -0,0 +1,146 @@
+from indico import IndicoClient
+from indico.queries import GraphQLRequest
+import json
+import string
+
+
+class ModelOp:
+
```

Comment:
It's nitpicking on code style, but I'd remove the extra newline here. And also rerun black on the file to make everything consistent.

```suggestion
```